app/assets/stylesheets/all/elmo_forms.css.scss

```diff
@@ -233,20 +233,10 @@ div.cascading-selects {
       border: none;
     }
 
-    div.label_and_control {
```

Comment:
Eeep! Changing elmo_forms styles globally makes me nervous. But if you've tested extensively and you're sure it works, then I'm ok with it. And also tested all the various modals?